pkg/cloudprovider/providers/gce/gce_clusters.go

```diff
@@ -37,21 +40,43 @@ func (gce *GCECloud) ListClusters(ctx context.Context) ([]string, error) {
 	return allClusters, nil
 }
 
+func (gce *GCECloud) GetClusters(ctx context.Context) ([]*container.Cluster, error) {
+	allClusters := []*container.Cluster{}
+	for _, zone := range gce.managedZones {
```

Comment:
`allClusters` seems misleading if it's just clusters in managed zones?

Reply:
Agreed and fixed.